app/src/main/java/org/mozilla/fenix/ext/Activity.kt

```diff
@@ -64,8 +65,14 @@ fun Activity.breadcrumb(
 
 /**
  * Opens Android's Manage Default Apps Settings if possible.
+ * Otherwise navigates to the Sumo article indicating why it couldn't open it.
+ * @param from fallback direction in case, couldn't open the setting.
```

Comment:
nit: In most places we have a `newline` in between description and params.

Reply:
👍🏽